README.md

```diff
@@ -1,4 +1,7 @@
 # user-data-for-fraud-prevention
+<!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
+[![All Contributors](https://img.shields.io/badge/all_contributors-1-orange.svg?style=flat-square)](#contributors-)
```

Comment:
@rachelquan Will this add the list of all contributors names as part of the badge? If so, I think it would be better to have this at the bottom of readme, so that the page does not look cluttered as the list grows. Does that make sense?

Reply:
Can you please add a screenshot of how this bade looks?

Reply:
![image](https://user-images.githubusercontent.com/17691502/94863375-448e8980-0432-11eb-9d29-02dc6b810b03.png)

You can check yourself by looking at the README in the branch for this PR https://github.com/rachelquan/user-data-for-fraud-prevention/tree/rachel-quan-allcontributors-bot

Reply:
@rachelquan Can you please add this badge next to all the other badges? Next to prettier badge?
<img width="477" alt="Screen Shot 2020-10-01 at 22 17 20" src="https://user-images.githubusercontent.com/4538858/94864563-17db7180-0434-11eb-9a60-e20181955dc0.png">

Reply:
Also, to add on, if there are more contributors, it will most likely look like the following README: https://github.com/all-contributors/all-contributors.

Reply:
@rachelquan Please fix the badge position and I will merge the PR. Thank you!

Reply:
![image](https://user-images.githubusercontent.com/39972689/94867438-1ee38f00-03f6-11eb-8763-7ec542c520ab.png)